ext/pcntl/pcntl.c

```diff
@@ -682,8 +690,27 @@ PHP_FUNCTION(pcntl_wifsignaled)
 }
 /* }}} */
 
-/* {{{ proto int pcntl_wexitstatus(int status) 
-   Returns the status code of a child's exit */
+
+/* {{{ proto bool pcntl_wifcontinued(int status) 
+   Returns true if the child status code represents a resumed process (WUNTRACED must have been used with waitpid) */
+PHP_FUNCTION(pcntl_wifcontinued)
+{
+#ifdef WIFCONTINUED
+        zend_long status_word;
+
+        if (zend_parse_parameters(ZEND_NUM_ARGS() TSRMLS_CC, "l", &status_word) == FAILURE) {
+               return;
+        }
+
+        if (WIFCONTINUED(status_word))
+                RETURN_TRUE;
+#endif
+        RETURN_FALSE;
```

Comment:
It looks like all the other functions above it do the same, so probably better to not throw a warning for consistency.